syntax = "proto3";
package example;
service FormatData {
      rpc DoFormat(Stock) returns (Rlt){}

}

message Stock {
      string direct = 1;
      string index_code = 2;
      string time = 3;
      float price = 4;
      int32 amount = 5;
      int32 type = 6;
}

message Rlt {
    string direct = 1;
    string index_code = 2;
    string time = 3;
    float price = 4;
    int32 amount = 5;
    int32 type = 6;
}
